rules/windows/discovery_net_view.toml

```diff
@@ -11,7 +11,39 @@ index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Windows Network Enumeration"
-note = """## Config
+note = """## Triage and analysis
+
+### Investigating Windows Network Enumeration
+
+After a successful initial compromise, attackers may try to gain situational awareness of the environment to plan their
+next steps. This can happen by running commands to enumerate network resources, users, connections, files, and security
+software installed.
+
+This rule looks for the execution of the `net` utility to enumerate servers in the environment that hosts shared drives
```

Comment:
```suggestion
This rule looks for the executed `net` utility that enumerates servers in the environment that hosts shared drives
```

Reply:
Feel free to ignore the second change if it's incorrect. Also note that I made this change throughout this PR and the same applies for all of those instances as well.